racket/src/ChezScheme/unicode/extract-info.ss

```diff
@@ -286,15 +291,17 @@
     (setprop i (fxand (getprop i) (fxnot constituent-property))))
   (commonize* tbl)
   (with-output-to-file* "unicode-charinfo.ss"
-    (lambda () 
+    (lambda ()
       (parameterize ([print-graph #t] [print-vector-length #f])
         (pretty-print
           `(module ($char-constituent? $char-subsequent? $char-upper-case? $char-lower-case? $char-title-case? $char-alphabetic?
                     $char-numeric? $char-whitespace? $char-cased? $char-case-ignorable? $char-category
                     $wb-aletter? $wb-numeric? $wb-katakana? $wb-extend? $wb-format? $wb-midnum? $wb-midletter?
                     $wb-midnumlet? $wb-extendnumlet? $char-combining-class $char-dump
                     ; UNICODE 7.0.0
-                    $wb-hebrew-letter? $wb-single-quote? $wb-double-quote? $wb-regional-indicator?)
+                    $wb-hebrew-letter? $wb-single-quote? $wb-double-quote? $wb-regional-indicator?
+                    ; UNICODE 14.0.0
+                    $wb-wsegspace? $wb-zwj?)
```

Comment:
These aren't used yet, so they probably shouldn't be in this PR. I thought I would need them to change the word boundary detection algorithm used in `string-titlecase` but ended up not, as I mentioned in a previous comment.

Both will be needed for [grapheme cluster boundary detection](https://unicode.org/reports/tr29/#Grapheme_Cluster_Boundary_Rules) however, as will an additional data file to provide the `Extended_Pictographic` character property.

Reply:
Sorry, that's not quite right. The `WSegSpace` property isn't used in the grapheme boundary rules, only the word boundary ones. So I suppose we don't need it at all (unless we want/need to implement the full word boundary detection algorithm).